### DISCUSSION TITLE
Ukrainian translation

### DISCUSSION THREAD

**Discussion by kovaroman** [vuejs-translations/guidelines#25] [General]
Here will be discussion about ukrainian translation of Vuejs 3 docs. Will be glad for any help in translation to ukrainian)

**Comment by nazar1ua** [comment]
Привіт, я хочу допомогти перекласти на українську мову

**Comment by Perryn000** [comment]
Також можу допомогти з українською. Маю гарний досвід з vue2

**Comment by NeoZis** [comment]
Можу долучитися до перекладу.

**Comment by winato** [comment]
Привіт, хочу допомогти. Маю досвід з Vue 2 та Vue 3.

**Comment by pawlek** [comment]
Долучусь до перекладу )

**Comment by maxdzin** [comment]
Привіт всім!
Хочу допомогти з перекладом на українську.
Маю багатсько досвіду з Vue v2 та v3.

**Comment by matrunchyk** [comment]
Привіт! Для тих, хто бажає долучитись до перекладу, запрошую сюди 

Репозиторій: https://github.com/vuejsorgua/docs
Телеграм-канал: https://t.me/vuejs_ukraine
Vue3 переклад: https://vuejs.org.ua/
Vue2 переклад: https://v2.vuejs.org.ua/

Дякую за підтримку!

**Reply by matrunchyk** [reply]
@NataliaTepluhina @yyx990803 Hello guys, do you mind publishing the Ukrainian translation on ua.vuejs.org? We would love to move from vuejs.org.ua to ua.vuejs.org instead to make it more official and potentially involve more translators.

For now, we are a small team of 5 active people who love the Vue ecosystem, and our wish is to have VueJS, Nuxt, Router, and Pinia localized to Ukrainian, which is, even more, demandable nowadays.

Thanks a lot!

**Reply by matrunchyk** [reply]
Kindly friendly reminder @NataliaTepluhina @yyx990803 :)

**Reply by yyx990803** [reply]
Hi @matrunchyk, sorry for the delay and thank you for the hard work!

To make the deployment management easier, we'd like to move the repo into this organization. I have sent you an invite - once accepted, you should be able to transfer the ua translation repo into this organization. Once that is done, we can setup official deployments.

Please also let me know what other members of the ua translation team you want to invite.

**Reply by matrunchyk** [reply]
I really appreciate it @yyx990803, thank you!

I transferred the repo but seems like I don't have permission to rename it to `docs-uk` (following the pattern). May I kindly ask you to do so (or give permission)? Also, it seems like the repo is private (I don't see it in the repo list in this organization).

Regarding team members, please invite @nazar1ua, @dddarkasss, @vozniukvolodymyr so that they can open PRs and keep maintaining the repo.

What a great achievement for us and great support for the whole Ukrainian community. And of course, this will allow us to bring more people to our translation community.

Thanks again.

**Reply by matrunchyk** [reply]
Also, seems like I cannot edit Wiki pages for our repo where we keep Glossary and Ukrainian community rules.

**Reply by yyx990803** [reply]
Yeah I didn't configure the team with permissions yet, now should be all good - also invited other members.

Note the repo is renamed to `docs-ua` - we use [ISO 639-1 language codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) so the code for Ukrainian is `ua` (hence also the team name)

**Reply by matrunchyk** [reply]
[As per ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes#:~:text=uig-,Ukrainian,-uk), it's `uk`, isn't it?
![image](https://user-images.githubusercontent.com/2089828/194849916-c7954f40-74ef-4db6-821a-5b1f13999cc6.png)

**Reply by yyx990803** [reply]
Oh I'm so sorry! I don't know why I got it wrong, I actually checked to make sure...

**Reply by yyx990803** [reply]
Regarding the domain name: we don't necessarily use the ISO code, it's a bit up to the translator's preference here (e.g. Simplified Chinese docs use `cn.vuejs.org` but Japanese v3 docs is `ja.vuejs.org`).

Do you prefer `ua.vuejs.org`?

**Reply by matrunchyk** [reply]
Yes, for the domain name let's use `ua`. Thanks!

**Reply by yyx990803** [reply]
https://ua.vuejs.org/ is live.

**Reply by matrunchyk** [reply]
Thanks mate!

**Reply by matrunchyk** [reply]
@yyx990803 I wonder why the repo is not visible on the organization page - the repo is public... Could you please take a look on your side?
